expression/explain.go

```diff
@@ -95,6 +95,15 @@ func ExplainExpressionList(exprs []Expression, schema *Schema) string {
 		switch expr.(type) {
 		case *Column, *CorrelatedColumn:
 			builder.WriteString(expr.String())
+		case *Constant:
+			v := expr.String()
+			if len(v) < 16 {
+				builder.WriteString(v)
+			} else {
+				builder.WriteString(fmt.Sprintf("%s(len:%d)", v[:16], len(v)))
```

Comment:
@winoros @XuHuaiyu PTAL, I'm not sure if this change is appropriate.

Reply:
I'm ok with this change. But the 16 may be a little small?

Reply:
How about 64?